docs/source/installation.rst

```diff
@@ -0,0 +1,18 @@
+Installation
+===========================================================================
+
+OML is available in PyPI:
+
+.. code-block:: console
+
+    pip install -U open-metric-learning
+
+You can also pull the prepared image from DockerHub or build one by your own
```

Comment:
split pull and build into several blocks

Reply:
ok